### ISSUE TITLE
Origin path error

### ISSUE THREAD

**Issue by ciaoben** [davidolrik/sublime-rsync-ssh#15]
Suddently, rsync has stopped working. It seems that there is a misconception of the host path. Here my conf:
```
{
	"folders":
	[
		{
			"follow_symlinks": true,
			"path": "/home/ben/Scrivania/qboxmail_panel-PROD"
		}
	],
	"settings":
	{
		"rsync_ssh":
		{
			"excludes":
			[
			],
			"options":
			[
			],
			"remotes":
			{
				"qboxmail_panel-PROD":
				[
					{
						"enabled": 1,
						"excludes":
						[
						 "tmp/",
						 "log/"
						],
						"options":
						[
						],
						"remote_host": "***",
						"remote_path": "/var/www/cbsolt/qboxmail_panel-PROD",
						"remote_port": 22,
						"remote_post_command": "",
						"remote_pre_command": "",
						"remote_user": "***"
					}
				]
			},
			"sync_on_save": true
		}
	}
}

```
It ha salways worked, but today, it doesn't work. I have opened the shell, and here is the command that create:
```
rsync -v -zar -e ssh -q -T -p 22 -o ConnectTimeout=10 /home/ben/Scrivania/qboxmail_panel-PROD/qboxmail_panel-PROD/ ben@109.168.999.999:/var/www/cbsolt/qboxmail_panel-PROD --exclude=.DS_Store --exclude=log/ --exclude=tmp/
```

As you can see, the folder qboxmail_panel-PROD is ducplicated in the url. But the pwd of the file is this:

/home/ben/Scrivania/qboxmail_panel-PROD

**Comment by ciaoben** [comment]
I Confirm the issue. I've put the files in a duplicate folder and everything works fine, so the problem is in the composition of the url I guess

**Comment by davidolrik** [comment]
Your folder path and remote doesn't match - they should either both be absolute, or both be relative to your .sublime-project file.

**Comment by davidolrik** [comment]
What is the path you your .sublime-project file?

**Comment by ciaoben** [comment]
The file is in the qboxmail-_panel-PROD folder..before I used "." and it worked, not doesn't! anyway I am using absolute paths now and everything work fine!


The documentation is not very clear regard this!


thanks!

**Comment by davidolrik** [comment]
I have just released [v1.4.1](https://github.com/davidolrik/sublime-rsync-ssh/releases/tag/v1.4.1) that fixes the folder to remote matching so they no longer have to be both be absolute/relative.
A remote called '.' is also supported - but only with single folder projects.
